src/app/Common/Services/Infrastructure/Models/ForgotPasswordModel.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Common.Services.Infrastructure.Models;
```

Comment:
Just a suggestion to move e-mail models all together with e-mail service to an `Email` subfolder. This way it's going to be obvious that this specific `Models` folder needed to be used only for email-models.